hudi-cli/src/main/java/org/apache/hudi/cli/HoodiePrintHelper.java

```diff
@@ -52,19 +52,16 @@ public static String print(String[] header, String[][] rows) {
    * @param rows List of rows
    * @return Serialized form for printing
    */
-  public static String print(TableHeader rowHeader,
-      Map<String, Function<Object, String>> fieldNameToConverterMap,
-      String sortByField, boolean isDescending, Integer limit, boolean headerOnly,
-      List<Comparable[]> rows) {
+  public static String print(TableHeader rowHeader, Map<String, Function<Object, String>> fieldNameToConverterMap,
+      String sortByField, boolean isDescending, Integer limit, boolean headerOnly, List<Comparable[]> rows) {
 
     if (headerOnly) {
       return HoodiePrintHelper.print(rowHeader);
     }
 
-    Table table = new Table(rowHeader, fieldNameToConverterMap,
-        Option.ofNullable(sortByField.isEmpty() ? null : sortByField),
-        Option.ofNullable(isDescending),
-        Option.ofNullable(limit <= 0 ? null : limit)).addAllRows(rows).flip();
+    Table table =
```

Comment:
kind of dislike that we are wasting the first line.. Can we fix it so that we use line 62 as well?

Reply:
This is split by spotless plugin, and the line length is defined to 120(org.eclipse.jdt.core.formatter.lineSplit) in eclipse-java-google-format.xml, we need to make org.eclipse.jdt.core.formatter.lineSplit a little larger than 120(such as 200?) to use the above line 62. But IMHO, 120 is a proper length and just let spotless plugin formats the code style, WDYT?

Reply:
Okay fair enough. We can leave it as is.